AppHandling/Run-AlPipeline.ps1

```diff
@@ -1760,7 +1806,7 @@ Write-Host -ForegroundColor Yellow @'
         if ($escapeFromCops) {
             Write-Host "Retrying without Cops"
             if ($useCompilerFolder) {
-                $appFile = Compile-AppWithBcCompilerFolder @parameters
+                $appFile = Invoke-Command -ScriptBlock $CompileAppWithBCCompilerFolder -ArgumentList $PreCompileApp, $PostCompileApp, $Parameters

```

Comment:
Not sure this is the right approach - this means that you might get pre and post called twice for the same app.
I think you should call pre before compiling - then compile and maybe without cops and then call post.

Reply:
Now the scripts are called only once.